libs/api/domains/endorsement-system/src/lib/dto/pagination.input.ts

```diff
@@ -0,0 +1,25 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString, IsNumber, IsOptional } from 'class-validator'
+
+@InputType()
+export class PaginationInput {
+  @Field({ nullable: true })
+  @IsNumber()
+  @IsOptional()
+  limit?: number
+
+  @Field({ nullable: true })
+  @IsString()
+  @IsOptional()
```

Comment:
These class validators don't do anything for the GraphQL schema and thus the validation.

Better to remove this since it creates noise.